README.md

```diff
@@ -711,3 +131,19 @@ There are two more connectors from the open-source community available:
 * [FZambia/tarantool](https://github.com/FZambia/tarantool).
 
```

Comment:
See feature comparison in the [documentation](https://www.tarantool.io/en/doc/latest/book/connectors/#go-feature-comparison).

Reply:
Updated.